lang/python/python-aiohttp/Makefile

```diff
@@ -0,0 +1,56 @@
+#
+# Copyright (C) 2019 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=aiohttp
+PKG_VERSION:=3.5.4
+PKG_RELEASE:=1
+
+PKG_SOURCE:=aiohttp-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/a/aiohttp/
+PKG_HASH:=9c4c83f4fa1938377da32bc2d59379025ceeee8e24b89f72fcbccd8ca22dc9bf
+
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-aiohttp-$(PKG_VERSION)
+
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILES:=LICENSE
+PKG_CPE_ID:=cpe:/a:aio-libs_project:aiohttp
+PKG_MAINTAINER:=Josef Schlehofer <pepe.schlehofer@gmail.com>
+
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+PKG_UNPACK:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+
+define Package/python3-aiohttp
+  SECTION:=lang
+  CATEGORY:=Languages
+  SUBMENU:=Python
+  TITLE:=Async http client/server framework (asyncio)
+  URL:=https://github.com/aio-libs/aiohttp
+  DEPENDS:= \
+	+PACKAGE_python3-aiohttp:python3-light \
+	+PACKAGE_python3-aiohttp:python3-attrs \
+	+PACKAGE_python3-aiohttp:python3-chardet \
+	+PACKAGE_python3-aiohttp:python3-multidict \
+	+PACKAGE_python3-aiohttp:python3-async-timeout \
+	+PACKAGE_python3-aiohttp:python3-yarl \
+	+PACKAGE_python3-aiohttp:python3-logging \
+	+PACKAGE_python3-aiohttp:python3-codecs \
+	+PACKAGE_python3-aiohttp:python3-cgi \
+	+PACKAGE_python3-aiohttp:python3-openssl
```

Comment:
`PACKAGE_python3-aiohttp` conditional also not necessary (since there is only the Python 3 package).

Reply:
Ok, ok.